file already exists exception against cache

```
j! -s faker@jbangdev -c "Stream.generate(faker.chuckNorris()::fact).limit(100).forEach(s -> println(s))" | jbang -s prelude@maxandersen -c '$$(l -> l.s(2))'
Exception in thread "main" dev.jbang.cli.ExitException: Resource could not be copied from class path: jbang.properties
	at dev.jbang.source.resolvers.ClasspathResourceResolver.getClasspathResource(ClasspathResourceResolver.java:66)
	at dev.jbang.source.resolvers.ClasspathResourceResolver.resolve(ClasspathResourceResolver.java:31)
	at dev.jbang.source.resolvers.CombinedResourceResolver.lambda$resolve$0(CombinedResourceResolver.java:26)
	at java.base/java.util.stream.ReferencePipeline$3$1.accept(ReferencePipeline.java:197)
	at java.base/java.util.Spliterators$ArraySpliterator.tryAdvance(Spliterators.java:1002)
	at java.base/java.util.stream.ReferencePipeline.forEachWithCancel(ReferencePipeline.java:129)
	at java.base/java.util.stream.AbstractPipeline.copyIntoWithCancel(AbstractPipeline.java:527)
	at java.base/java.util.stream.AbstractPipeline.copyInto(AbstractPipeline.java:513)
	at java.base/java.util.stream.AbstractPipeline.wrapAndCopyInto(AbstractPipeline.java:499)
	at java.base/java.util.stream.FindOps$FindOp.evaluateSequential(FindOps.java:150)
	at java.base/java.util.stream.AbstractPipeline.evaluate(AbstractPipeline.java:234)
	at java.base/java.util.stream.ReferencePipeline.findFirst(ReferencePipeline.java:647)
	at dev.jbang.source.resolvers.CombinedResourceResolver.resolve(CombinedResourceResolver.java:28)
	at dev.jbang.source.ResourceRef.forResource(ResourceRef.java:136)
	at dev.jbang.Configuration.getBuiltin(Configuration.java:265)
	at dev.jbang.Configuration.defaults(Configuration.java:218)
	at dev.jbang.Configuration.getMerged(Configuration.java:251)
	at dev.jbang.Configuration.instance(Configuration.java:196)
	at dev.jbang.cli.JBang$ConfigurationResourceBundle.getKeys(JBang.java:212)
	at picocli.CommandLine$Model$Messages.keys(CommandLine.java:11250)
	at picocli.CommandLine$Model$Messages.<init>(CommandLine.java:11234)
	at picocli.CommandLine$Model$CommandSpec.setBundle(CommandLine.java:6346)
	at picocli.CommandLine$Model$CommandSpec.resourceBundle(CommandLine.java:6342)
	at picocli.CommandLine.setResourceBundle(CommandLine.java:3319)
	at dev.jbang.cli.JBang.getCommandLine(JBang.java:227)
	at dev.jbang.cli.JBang.getCommandLine(JBang.java:107)
	at dev.jbang.Main.main(Main.java:12)
Caused by: java.nio.file.FileAlreadyExistsException: /Users/max/.jbang/cache/urls/0170b212db2e51c073685a14cf2df41f73ab803ee2461d7fcdbd83eba79664a4
	at java.base/sun.nio.fs.UnixException.translateToIOException(UnixException.java:94)
	at java.base/sun.nio.fs.UnixException.rethrowAsIOException(UnixException.java:106)
	at java.base/sun.nio.fs.UnixException.rethrowAsIOException(UnixException.java:111)
	at java.base/sun.nio.fs.UnixFileSystemProvider.newByteChannel(UnixFileSystemProvider.java:219)
	at java.base/java.nio.file.spi.FileSystemProvider.newOutputStream(FileSystemProvider.java:484)
	at java.base/java.nio.file.Files.newOutputStream(Files.java:228)
	at java.base/java.nio.file.Files.copy(Files.java:3160)
	at dev.jbang.source.resolvers.ClasspathResourceResolver.getClasspathResource(ClasspathResourceResolver.java:63)
	... 26 more
WARNING: Using incubator modules: jdk.incubator.foreign, jdk.incubator.vector
```

looks like a copy is not overwriting or even copying when it should not.

--- a/src/main/java/dev/jbang/source/resolvers/ClasspathResourceResolver.java
+++ b/src/main/java/dev/jbang/source/resolvers/ClasspathResourceResolver.java
@@ -7,7 +7,6 @@ import java.net.URISyntaxException;
 import java.net.URL;
 import java.nio.file.Files;
 import java.nio.file.Path;
-import java.nio.file.StandardCopyOption;
 
 import dev.jbang.cli.BaseCommand;
 import dev.jbang.cli.ExitException;
@@ -57,14 +56,18 @@ public class ClasspathResourceResolver implements ResourceResolver {
 		}
 
 		// We couldn't read the file directly from the class path so let's make a copy
-		try (InputStream is = url.openStream()) {
-			Path to = Util.getUrlCache(cpResource);
-			Files.createDirectories(to.getParent());
-			Files.copy(is, to, StandardCopyOption.REPLACE_EXISTING);
-			return ResourceRef.forCachedResource(cpResource, to.toFile());
-		} catch (IOException e) {
-			throw new ExitException(BaseCommand.EXIT_GENERIC_ERROR,
-					"Resource could not be copied from class path: " + ref, e);
+		Path to = Util.getUrlCache(cpResource);
+		if (!Files.exists(to)) {
+			try (InputStream is = url.openStream()) {
+				Files.createDirectories(to.getParent());
+				Files.copy(is, to);
+			} catch (IOException e) {
+				Util.deletePath(to, true);
+				throw new ExitException(BaseCommand.EXIT_GENERIC_ERROR,
+						"Resource could not be copied from class path: " + ref, e);
+			}
 		}
+
+		return ResourceRef.forCachedResource(cpResource, to.toFile());
 	}
 }

